.evergreen/run-enterprise-auth-tests.sh

```diff
@@ -23,5 +23,10 @@ echo "Install dependencies"
 export BUNDLE_GEMFILE=gemfiles/mongo_kerberos.gemfile
 bundle install --gemfile="$BUNDLE_GEMFILE"
 
+unset CI
```

Comment:
When this env variable is set, `lite_spec_helper.rb` sets the formatter as `JsonExtFormatter`, which causes a `NoMethodError: undefined method `seed' for #<JsonExtFormatter:0x0000000002ec4f58>` error when rspec tries to output anything. I'm guessing there's another gem I have to require to fix this, but I'm not sure which.